### ISSUE TITLE
Problem : Session is global and should be avoided

### ISSUE THREAD

**Issue by Fen747** [Blockrazor/blockrazor#71]
Session is known for being a global variable, which is something that's to be deprecated in general.
Session in particular can be replaced by ReactiveDict / ReactiveVars.

As much as possible, variables should be scoped to the lower lever possible, to mimic react's state is a good thing for exemple.

```
Template.tmp.onCreated(function(){
    this.someDataSet = new ReactiveDict();
    this.smthgReactive = new ReactiveVar();
});
```

**Comment by anbud** [comment]
I'll take this and convert Sessions to ReactiveVars, it improves performance and stops the pollution of the global namespace.

**Comment by gazhayes** [comment]
Once you are sure it's fixed mention it here and then (usually) whoever opened the issue can check if they also think its fixed and close the issue (or if there's radio silence for a week or so you can close it yourself)

**Comment by anbud** [comment]
Yap, this is fixed in #111, there's no more Session usage in the codebase. I've replaced it everywhere with either ReactiveVar or ReactiveDict.
